src/components/Dialog/Dialog.js

```diff
@@ -58,7 +58,7 @@ const Header = styled.div`
 `;
 
 const Title = styled.span`
-  font-size: ${props => props.theme.fontSizes.normal};
+  font-size: ${props => props.theme.fontSizes.large};
   font-weight: bold;
```

Comment:
Maybe we want to drop this line as well to make it more similar to spec in #493 and also Material UI?

#### Bold

![image](https://user-images.githubusercontent.com/1216874/69859555-a91e0c80-1294-11ea-9692-14e96908d6ac.png)

#### Normal

![image](https://user-images.githubusercontent.com/1216874/69859584-b935ec00-1294-11ea-9796-d1e3aa9a8d0c.png)

Reply:
Thanks @fbarl, I agree!